website/docs/docs/cloud/manage-access/auth0-migration.md

```diff
@@ -54,9 +54,9 @@ After the configuration is saved, your SAML settings will look something like th
 
 Once you have saved this information in the SSO environment, you must edit some information in the dbt Cloud settings. Navigate to the **Account Settings**, update the single sign-on URL fields, and provide the updated x.509 certificate.
 
-Make sure to enable the "Use Auth0" option to ensure that the traffic is routed correctly. _The Auth0 migration action is final and cannot be undone_
+Toggle the `Enable new SSO authentication` option to ensure the traffic is routed correctly. _The Auth0 migration action is final and cannot be undone_
```

Comment:
@matthewshaver Should we change this to match the UI too? Maybe "changing your SSO configuration..."
 > The Auth0 migration action